zcash_client_backend/src/zip321.rs

```diff
@@ -651,29 +651,24 @@ mod parse {
                 )),
```

Comment:
Nit: the `digit0` above doesn't appear to match the ABNF in ZIP 321. Open an issue if not addressed here.

Reply:
Yes it should be `digit1`.